lib/removeWhiteSpace.js

```diff
@@ -0,0 +1,18 @@
+function removeWhiteSpace(str) {
```

Comment:
This function is not bundled correctly with babel, so either you move this function in the `index.js` file or you add something like `rollup` to this project.

Reply:
Or use CommonJS module. TBD what is the best solution